src/pages/profile/profile.scss

```diff
@@ -18,5 +18,69 @@ page-profile {
     .name {
         font-size: 30px;
     }
+    .section {
```

Comment:
These profile style updates should be in their own commit. I'll allow it this one time, but your issues/pr should be separate going forward.